include/rocksdb/iterator.h

```diff
@@ -108,6 +108,11 @@ class Iterator : public Cleanable {
   //   Get the user-key portion of the internal key at which the iteration
   //   stopped.
   virtual Status GetProperty(std::string prop_name, std::string* prop);
+
+  virtual Slice utimestamp() const {
+    assert(false);
+    return Slice();
+  }
```

Comment:
Reading the interface, I'm not very clear when calling Seek(), a user should use key with timestamp or not.

It may help to write some example code in code comments, as what is suggested in https://google.github.io/styleguide/cppguide.html#Function_Comments

Reply:
Will add comment. Seek(target), target does not contain timestamp. Timestamp is specified in read_options.